.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Build
+
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Build
+        run: |
+          docker build -t builder .
+      - name: Run
+        run: |
+          docker run --rm -v $PWD/dist:/dist -e BRANCH_NAME=${GITHUB_REF##*/} builder
+      - name: Create a release
+        if: startsWith(github.ref, 'refs/tags')
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+      - name: Upload binaries to release
+        uses: svenstaro/upload-release-action@v2
```

Comment:
Is the wildcard the primary  motivation for using this rather than https://github.com/actions/upload-release-asset?

Reply:
yes since we might have different types of artifacts to push.

Reply:
We have few repositories following the same patterns. (debian, centos, ubuntu) we could do with having a template.

Reply:
https://github.com/actions/upload-release-asset/issues/47